### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bjornsen**
Remove use of --address 0.0.0.0 and extraneous port comments.

### CODE REVIEW

File: ray-on-gke/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ To submit a Ray job, first establish a connection to the Ray head. For this exam
 to connect to the Ray head via localhost.
 
 ```bash
-$ kubectl port-forward --address 0.0.0.0 service/example-cluster-kuberay-head-svc 8265:8265 &
+$ kubectl port-forward service/example-cluster-kuberay-head-svc 8265 -n ml &
```
Comment (andrewsykim) [inline ray-on-gke/README.md:42]: nit: can you put the `-n` in front of `kubectl`

```
kubectl -n ml port-forward service/example-cluster-kuberay-head-svc 8265 -n ml & 
```

**Review by andrewsykim — APPROVED** [review]

File: ray-on-gke/README.md (hunk 1 above)
Comment (bjornsen) [reply ray-on-gke/README.md]: Done.

**Pull request merged**